.readthedocs.yaml

```diff
@@ -8,9 +8,9 @@ version: 2
 
 # Set the version of Python and other tools you might need
 build:
-  os: ubuntu-22.04
+  os: ubuntu-24.04
   tools:
-    python: mambaforge-22.9
+    python: mambaforge-23.11
```

Comment:
Update to the most recent documentation build environments. These should be closer to the environments we use locally, and in the nightly builds.